doc/planar.xml

```diff
@@ -0,0 +1,314 @@
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> KuratowskiPlanarSubdigraph(D);
+fail
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> IsPlanarDigraph(D);
+false
+gap> KuratowskiPlanarSubdigraph(D);
+[ [ 2, 9, 7 ], [ 3 ], [ 6 ], [ 5, 9 ], [ 6 ], [  ], [ 4 ], 
+  [ 7, 9, 3 ], [  ], [  ] ]
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="KuratowskiOuterPlanarSubdigraph">
+<ManSection>
+  <Attr Name="KuratowskiOuterPlanarSubdigraph" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    <C>KuratowskiOuterPlanarSubdigraph</C> returns the list of out-neighbours
+    of a (not necessarily induced) subdigraph of the digraph <A>digraph</A>
+    that witness the fact that <A>digraph</A> is not outer planar, or
+    <K>fail</K> if <A>digraph</A> is outer planar.  In other words,
+    <C>KuratowskiOuterPlanarSubdigraph</C> returns the out-neighbours of a
+    subdigraph of <A>digraph</A> that is homeomorphic to the complete graph
+    with four vertices, or to the complete bipartite graph with vertex sets of
+    sizes <C>2</C> and <C>3</C>. <P/>
+    
+    The directions and multiplicities of any edges in <A>digraph</A> are
+    ignored when considering whether or not <A>digraph</A> is outer planar. 
+    <P/>
+
+    See also 
+    <Ref Prop="IsOuterPlanarDigraph"/>,
+    <Ref Attr="SubdigraphHomeomorphicToK4"/>, and 
+    <Ref Attr="SubdigraphHomeomorphicToK23"/>.
+    <P/>
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> KuratowskiOuterPlanarSubdigraph(D);
+[ [ 3, 5, 10 ], [ 9, 8, 10 ], [ 4 ], [ 6 ], [ 11 ], [ 7 ], [ 8 ], 
+  [  ], [ 11 ], [  ], [  ] ]
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> IsOuterPlanarDigraph(D);
+false
+gap> KuratowskiOuterPlanarSubdigraph(D);
+[ [  ], [  ], [  ], [ 8, 9 ], [  ], [  ], [ 9, 4 ], [ 7, 9 ], [  ], 
+  [  ] ]
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="PlanarEmbedding">
+<ManSection>
+  <Attr Name="PlanarEmbedding" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a planar digraph, then 
```

Comment:
I'm not sure this documentation properly conveys what's happening here. The point is that if the vertices are drawn as suggested by the returned list, then you can fill in the edges without edges crossing. I feel like the connection between this clockwise property and planarity could be elucidated a bit more.

Reply:
Perhaps this could be clearer, I have not been able to find any description of the return value of this method (from the edge-addition-planarity-suite), so I basically copied what is written here from the corresponding manual entry from Sage see [here](https://doc.sagemath.org/html/en/reference/graphs/sage/graphs/generic_graph.html#sage.graphs.generic_graph.GenericGraph.get_embedding).